### DISCUSSION TITLE
Changing Themes Creates Issues

### DISCUSSION THREAD

**Discussion by JNewstedt** [OrchardCMS/OrchardCore#16868] [Q&A]
Why does changing a theme from the initial choice create design/CSS issues? It looks like a style script isn't loading after clicking "Make Current" to change a site's theme. 

Original Blog Recipe choice during build (looks fine):
![1](https://github.com/user-attachments/assets/8b0dc69e-d290-417d-99ef-38aa9641561b)

After switching to Agency theme:
![2](https://github.com/user-attachments/assets/b9906920-09d2-4949-9588-95a6f3eb1d81)

**Comment by hishamco** [comment]
Please steps to reproduce. but I'm assuming you changed the theme for a view not prepared well for the current theme

**Comment by Piedone** [comment]
Not all themes are suitable for all sites. The Blog recipe sets up a blog and the Blog theme, all tailored to present a simple blog. The Agency theme and its recipe, in contrast, are geared towards a portfolio website.

So, e.g. the Agency theme doesn't have styling for the banner image configured by the Blog recipe.

To put it otherwise, there can be a dependency between the content and configuration of the site and the theme. There doesn't have to be though, e.g. the Default Theme is very generic. But due to this, it's a basic theme, and lacks the specialization of the Blog or Agency themes.

**Reply by JNewstedt** [reply]
Am I to assume the option to switch themes doesn't actually have the ability to do so? I think there is more to it than just clicking that option, but it makes one wonder why the option is presented as a simple button.

**Reply by Piedone** [reply]
It does switch the theme, as you also observed. But it's like a phone case: you can switch the case freely as you wish, but while some cases will fit your phone perfectly, some will be slightly off, or usable only with compromises, and then there will be ones that won't fit at all.

**Reply by JNewstedt** [reply]
If this is the case, then what else needs to be done to make the new theme look as intended?

**Reply by Piedone** [reply]
Please check out the theming tutorials here, from video 20: https://www.youtube.com/playlist?list=PLuskKJW0FhJcSX7j0Bd-1X5hq3dgCtYwO
